dsl/src/commonMain/kotlin/it/unibo/collektive/field/FieldManipulation.kt

```diff
@@ -1,6 +1,9 @@
+@file:Suppress("TooManyFunctions")
```

Comment:
I think it is right. We should split these by type and change the file name

Reply:
I've moved the field operations into a type-based file name.
I've also added the missing operator for numeric types (`div`, `times`, and `rem`).
However,
For the `NumericField` file, we exceed the 11 functions triggering detekt.
For this case, the suppress warning is required since the operation and the type are coherent.
What do you think?

Reply:
ok